src/www/ui/api/Helper/DbHelper.php

```diff
@@ -491,7 +491,7 @@ public function insertNewTokenKey($userId, $expire, $scope, $name, $key)
    * Adds new oauth client to the user.
    *
    * @param string  $name     Name of the new client
-   * @param integer $userId   User PK
+   * @param integer￼ $userId   User PK
```

Comment:
Irrelevant change. Please remove.

Reply:
Thank you @GMishx , irrelevant change is now removed.